app/views/users/_customization.html.erb

```diff
@@ -78,20 +80,15 @@
       </p>
     </header>
 
-    <%= form_for(@user, html: { id: nil, class: "grid gap-4" }) do |f| %>
```

Comment:
removing nested forms that didn't work

Reply:
refactor (potentially blocking): I haven't pulled this locally yet, but most likely the nested form will need to be replaced with another element so that the CSS classes `grid gap-4` remain for layout. A [\<fieldset /\>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) should do the trick.

Reply:
Thanks @nickytonline. I merged it to unblock the functionality. I'll segue with a `<fieldset>`, thanks a lot for the review!